### PR TITLE
[Build Transaction] Basic Operations layout + button actions

### PR DESCRIPTION
**Pull request by quietbits**
- Added "Clear Params" button to clear entered form data on the Params page.
- Extended `Box` component to use it like flexbox. We often use flexbox, so this should make it easier to style UI.
- Basic layout for the Operations with actions to add, move, delete, and duplicate.

image

### CODE REVIEW

File: src/components/layout/Box/index.tsx [hunk 1]
```diff
@@ -6,14 +6,37 @@ export const Box = ({
...
+  direction?: "column" | "row" | "column-reverse" | "row-reverse";
+  justify?:
+    | "center"
+    | "space-between"
+    | "space-around"
+    | "end"
+    | "left"
+    | "right"
+    | "baseline";
+  align?: "center" | "end" | "start" | "baseline" | "stretch";
```
Comment (quietbits) [inline src/components/layout/Box/index.tsx:27]: New flexbox props to make building UI easier.

File: src/helpers/arrayItem.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  itemArray.splice(moveTo, 0, el);
+  return itemArray;
+};
+
+export const arrayItem = {
+  add: addItem,
+  delete: deleteItem,
+  duplicate: duplicateItem,
+  move: moveItem,
+};
```
Comment (quietbits) [inline src/helpers/arrayItem.ts:70]: Helper methods to modify array for Operations actions.

File: src/components/TabbedButtons/index.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  isDisabled?: boolean;
+  onClick: () => void;
+};
+
+type TabbedButtonsProps = {
+  size: "sm" | "md" | "lg";
+  buttons: TabbedButton[];
+};
+
+export const TabbedButtons = ({ size, buttons }: TabbedButtonsProps) => {
```
Comment (quietbits) [inline src/components/TabbedButtons/index.tsx:20]: We might move this component to SDS later.

**Review by jeesunikim — APPROVED** [review]

File: src/helpers/arrayItem.ts (hunk 2 above)
Comment (jeesunikim) [reply src/helpers/arrayItem.ts:70]: I can use the helper function 'add' for adding an additional signature! 🎉

**Pull request merged**